tests/general_generators.py

```diff
@@ -33,6 +33,10 @@ def any_past_datetime_string() -> str:
     return any_past_datetime().isoformat()
 
 
+def any_past_datetime_string_without_tz() -> str:
+    return any_past_datetime().strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
The "Z" part of the string is a timezone, so isn't this *with* time zone?

Reply:
I think I meant no offset. Pystac was formatting datetimes like these and I was trying to align them.
would any_past_datetime_string_without_offset() be ok?

Reply:
I'd probably use "UTC" explicitly in the name rather than "without offset", so something like `any_past_utc_datetime_string`.